### DISCUSSION TITLE
Tree generator error

### DISCUSSION THREAD

**Discussion by FPYS** [UMEP-dev/UMEP#298] [Q&A] [answered]
Hi,

I get the following error trying to use the tree generator : 

ValueError: zero-size array to reduction operation maximum which has no identity

Here are my files : 

[Carto.zip](https://github.com/UMEP-dev/UMEP/files/6726657/Carto.zip)

Thanks :)

**Comment by biglimp** [comment]
@nilswallenberg , can you have a look?

**Comment by nilswallenberg** [comment, marked as answer]
Hi FPYS,

Correct me if I'm wrong: 
type = tree type 
Cironf_M = diameter of tree 
TrunkH = trunk zone height
hauteurenm = tree height

I think the reason why you are getting this error is because the resolution of your raster's is 2 meters, whereas the diameters of some of the trees is smaller than this. 

What you can do is either change the resolution of your raster's to a higher resolution, or set all tree diameters to same diameter as the raster resolution (that is 2 meter), because now everything below 2 will become zero.

Best,
Nils

**Reply by FPYS** [reply]
Thanks, it worked :)
